app/controllers/bookings_controller.rb

```diff
@@ -1,11 +1,22 @@
 class BookingsController < ApplicationController
+  before_action :current_booking, only: [:show, :edit, :update, :destroy]
   def index
+    @bookings = Booking.all
```

Comment:
this will need pundit, only meant to be accessed by an admin user, we'll have to make restrictions.
Can you add a comment so we remember to update it accordingly?